_posts/2021-12-31-a-year-in-review-2021.markdown

```diff
@@ -0,0 +1,65 @@
+---
+layout: epic
+title: "A Year in Review: 2021"
+date: 2022-01-06
+categories: [team, engineering]
+author: artsy_engineering
+comment_id: 714
+---
+
+Ufff, 2021 was a tough year. It was the second year of the Covid-19 pandemic. We tried our best to support each
+other through these challenging times and in that spirit, we feel like it's important not to lose sight of our
+accomplishments. As we say goodbye to 2021 and hello to 2022, here are some Artsy Engineering wins from this past year.
+
+What did you ship in 2021? We would love to hear about it and celebrate with you!
+
+<!-- more -->
+
+- Recorded and released 42 episodes of the [Artsy Engineering Radio][]
+- Onboarded 23 engineers in Berlin, Germany
+- Leveraged our investment in React Native to launch the [Artsy Android App][]
+- We [opened][] 13,368 pull requests! 12,054 were [merged][] and 1,050 were [closed][]. (note: includes contributions in private repositories)
+- Responded to and resolved 29 incidents
+- Collaborated with [iTechArt][] engineers on impactful projects
+- Migrated more repositories to deploy continuously
+- Significantly improved [Core Web Vitals][] across our web surfaces
+- Answered a bunch of questions in our #dev-help Slack channel to help get each other unstuck!
+- After many false starts, we finally agreed to pull the trigger on a full auction bidding engine rewrite
+- More stable end-to-end automated testing tooling (codenamed "integrity")
+- Worked with [Applause][] to help us with external QA for our mobile apps
+- Bootstrapped a new data engineering team
+- Launched our a new recommendation model (codenamed "affinity score")
+- Developed an in-house auction results pipeline
+- Launched improvements to our recommendations (New Works for You) for the first time in a while!
+- Launched integrations with [Arta][] (shipping), [Avalara][] (tax compliance), [Algolia][] (search), [Adjust][] (analytics), [Braze][] (marketing), and [Postie][] (marketing).
+- Moved 38+ of Force apps to a new and improved V2 setup
```

Comment:
```suggestion
- Moved 38+ of [Force][] apps to a new and improved V2 setup
```

as a non-artsy person, i guess that would be a helpful link

Reply:
Addressed in https://github.com/artsy/artsy.github.io/pull/706/commits/3d46743805d457e85ba20ef940fa4a32d99005e0. Thanks!